src/main/java/com/d2fn/guano/RestoreJob.java

```diff
@@ -55,16 +53,12 @@ public void go() {
      */
     @Override
     public void visit(File f, byte[] data, String znode) {
-//        System.out.println(f.getPath());
-//        System.out.println(" -> " + znode);
         createOrSetZnode(data, znode);
     }
 
     private void createOrSetZnode(byte[] data, String znode) {
         try {
             String s = zk.create(znode, data, ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
-        } catch (KeeperException.NoNodeException e) {
-            e.printStackTrace(System.err);
         } catch (KeeperException.NodeExistsException e) {
```

Comment:
Do we not need to still handle the `NoNodeException`?  Based on what I see below I would have expected it to be "or"'d in via a `|`?  or is this one of the "others have redundant logic." cases?

Reply:
A `KeeperException.NoNodeException` extends `KeeperException` which was already being caught below, and both blocks were just printing the stacktrace to `System.err`